draft-ietf-acme-acme.md

```diff
@@ -760,11 +760,19 @@ notAfter (optional, string):
 : The requested value of the notAfter field in the certificate, in the date
 format defined in {{!RFC3339}}
 
-requirements (required, array):
-: The requirements that the client needs to fulfill before the requested
-certificate can be granted (for pending applications).  For final applications,
-the requirements that were met.  Each entry is a dictionary with parameters
-describing the requirement (see below).
+authorizations (required, array):
+: For pending applications, the authorizations that the client needs to complete
+before the requested certificate can be granted (see
```

Comment:
I think changing "granted" -> "issued" would better match the wording used elsewhere.